tests/test_deck.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+
+from deck import Deck
+
+
+class DeckTestCase(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.deck = Deck()
+
+    def test_deck_for_number_of_cards(self):
+        self.assertEqual(len(self.deck.cards), 52)
+
+    def test_deck_for_duplicates(self):
+        no_dupes = set(self.deck.cards)
```

Comment:
I don't think this would actually check that there are no dupes. This would work for something like integers or strings. But not for custom classes. We can make it work for custom classes, but in order to do so, we'll need to add `__eq__` and `__hash__` to the class I think. I don't think I've ever actually done this before, but I'm pretty sure it would work then.

`set` was a clever solution. Just needs a little bit of extra stuff to make it actually work.

Reply:
Ooh, that's an oversight. I think I fixed it by only adding `__hash__`. I checked it by adding a duplicate of the first card in the deck -- It did fail.

Also, I don't know if there was a better way to build `__hash__` for `Card`... It feels like there might be something else I could've done.